test/test_torch.py

```diff
@@ -2440,6 +2440,21 @@ def _generate_noncontiguous_input():
                     self.assertEqual(tensor.repeat(*repeat).numpy(),
                                      np.tile(tensor.numpy(), repeat))
 
+        @unittest.skipIf(not TEST_NUMPY, "Numpy not found")
+        def test_tile(self):
```

Comment:
This test should be device generic and placed in TestTorchDeviceType. See those tests for how they take a device argument.